clients/include/blas1/testing_axpy_batched.hpp

```diff
@@ -159,34 +162,37 @@ void testing_axpy_batched(const Arguments& arg)
                     HIPBLAS
         =================================================================== */
         CHECK_HIPBLAS_ERROR(hipblasSetPointerMode(handle, HIPBLAS_POINTER_MODE_DEVICE));
-        CHECK_HIPBLAS_ERROR(hipblasAxpyBatchedFn(handle,
-                                                 N,
-                                                 d_alpha,
-                                                 dx.ptr_on_device(),
-                                                 incx,
-                                                 dy_device.ptr_on_device(),
-                                                 incy,
-                                                 batch_count));
+        DAPI_CHECK(hipblasAxpyBatchedFn,
+                   (handle,
+                    N,
+                    d_alpha,
+                    dx.ptr_on_device(),
+                    incx,
+                    dy_device.ptr_on_device(),
+                    incy,
+                    batch_count));
 
         CHECK_HIPBLAS_ERROR(hipblasSetPointerMode(handle, HIPBLAS_POINTER_MODE_HOST));
-        CHECK_HIPBLAS_ERROR(hipblasAxpyBatchedFn(handle,
-                                                 N,
-                                                 &alpha,
-                                                 dx.ptr_on_device(),
-                                                 incx,
-                                                 dy_host.ptr_on_device(),
-                                                 incy,
-                                                 batch_count));
+        DAPI_CHECK(hipblasAxpyBatchedFn,
+                   (handle,
+                    N,
+                    &alpha,
+                    dx.ptr_on_device(),
+                    incx,
+                    dy_host.ptr_on_device(),
+                    incy,
+                    batch_count));
 
         CHECK_HIP_ERROR(hy_host.transfer_from(dy_host));
         CHECK_HIP_ERROR(hy_device.transfer_from(dy_device));
 
         /* =====================================================================
                     CPU BLAS
         =================================================================== */
-        for(int b = 0; b < batch_count; b++)
+        for(int64_t b = 0; b < batch_count; b++)
         {
-            ref_axpy<T>(N, alpha, hx_cpu[b], incx, hy_cpu[b], incy);
+            int b2 = b;
+            ref_axpy<T>((int)N, alpha, hx_cpu[b2], (int)incx, hy_cpu[b2], (int)incy);
```

Comment:
Did it error here without the cast?

Reply:
Just compiler warnings, didn't try running